build-aux/snap/snapcraft.yaml

```diff
@@ -0,0 +1,92 @@
+name: ddclient
+summary: Ddclient updates dynamic DNS entries on a wide range of dynamic DNS services
+description: |
+  ddclient is a Perl client used to update dynamic DNS entries
+  for accounts on many dynamic DNS services. It uses curl for internet access.
+  Full list of the supported services can be found at the project page:
+  https://github.com/ddclient/ddclient?tab=readme-ov-file
+
+  Once installed, configure config file and enable service with
+  `$ snap start --enable ddclient.daemon`
+  For more information run `$ ddclient`
+
+type: app
+base: core24
+adopt-info: ddclient
+license: GPL-2.0
+
+platforms:
+  amd64:
+  armhf:
+  arm64:
+
+confinement: strict
+grade: stable
+
+environment:
+  LD_LIBRARY_PATH: ${SNAP}/usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}
+  PATH: ${SNAP}/usr/bin:/usr/bin
+  PERL5LIB: ${SNAP}/usr/share/perl:${SNAP}/usr/share/perl5:${SNAP}/usr/share/perl-openssl-defaults:${SNAP}/usr/lib/${CRAFT_ARCH_TRIPLET_BUILD_FOR}/perl
+
+apps:
+  ddclient:
+    command: usr/bin/ddclient-wrapper --help
+
+  daemon:
+    command: usr/bin/ddclient-wrapper
+    install-mode: disable
+    daemon: simple
+    plugs:
+      - network
+
+parts:
+  ddclient:
+    plugin: autotools
+    source: .
+    autotools-configure-parameters:
+      - --prefix=/usr
+      - --sysconfdir=/etc/ddclient
+      - --localstatedir=/var
```

Comment:
I would expect the autotools plugin to pass appropriate defaults for these parameters.  Does it not?